docs/12-performance-test.md

```diff
@@ -3,84 +3,64 @@ Performance tests
-```
-
-Find the ECS cluster that is running the containers that are writing data for ingest. It will have the name
-`sleeper-${ID}-system-test-cluster`. There should be 11 running tasks. Wait until all those tasks have finished.
-Click on one of the tasks and find the corresponding Cloudwatch log group. Use the following command in
-Cloudwatch Log Insights with the above log group selected:
-
-```
-fields @message 
-| filter @message like "to S3"
-# If you want to see all the results, comment out the next two lines
-| parse @message '* * - * at * per second' as class, level, msg, rate
-| stats avg(rate)
+SUBNETS=<ids-of-the-subnets-to-deploy-to>
+./scripts/test/compactionPerformance/buildDeployTest.sh ${ID} ${VPC} ${SUBNETS}
 ```
 
-This should result in a single value which summarises the performance of the containers that are ingesting data
-(these are direct standard ingest tasks, i.e. they are writing data directly to Sleeper using the standard ingest
-approach - they are not using the ingest queue). The table below records the performance for various versions of
-Sleeper.
+Report the results:
 
-Now find the ECS cluster that runs compaction tasks. It will be named `sleeper-${ID}-merge-compaction-cluster`.
-Click on one of the tasks and find the corresponding Cloudwatch log group. Use the following command in Cloudwatch
-Log Insights with the above log group selected:
+```bash
+# Ingest performance figures can be found by running the following reports
+./scripts/utility/ingestTaskStatusReport.sh ${ID} standard -a
+./scripts/utility/ingestJobStatusReport.sh ${ID} system-test standard -a
 
+# Compaction performance figures can be found by running the following reports
+./scripts/utility/compactionTaskStatusReport.sh ${ID} standard -a
+./scripts/utility/compactionJobStatusReport.sh ${ID} system-test standard -a
 ```
-fields @message | filter @message like "compaction read"
-# If you want to see all the results, comment out the next two lines
-| parse @message '* * - * at * per second' as class, level, msg, rate
-| stats avg(rate)
-```
-
-This should result a single value summarising the performance of the containers that are performing a compaction. See
-the table below for the results for various versions of Sleeper.
-
-## Compaction performance
 
-This test will continue running and wait for each operation in the tests to run. This will take around an hour. This
-can be used to measure performance of compaction with a fixed order of partition splitting and compaction job creation.
-This is intended to avoid any variance that may be caused by the number of input files or the amount of data processed
-at once.
+## Bulk import performance
+This will test the performance of bulk import using EMR. This will take about 30 minutes. A bulk import job for 
+1 billion records will be created and sent to the bulk import queue 5 times. It will then wait for all EMR steps to finish, 
+then check that there are 5 billion records in the table.
```

Comment:
I think I like the impact if you put the time it takes at the end, right after you said it's 5 billion records. It probably is better to say it up front though :(